docs/LAYERS.md

```diff
@@ -0,0 +1,287 @@
+# OpenMapTiles - Schema, Layer & Tileset explanation  
+OpenMapTiles is an extensible and open vector tile schema for a OpenStreetMap basemap. It is used to generate vector tiles for [openmaptiles.org](https://openmaptiles.org/).    
+        
+- :link: Docs https://openmaptiles.org/docs    
+- :link: Schema: https://openmaptiles.org/schema    
+- :link: Production package: https://openmaptiles.com/
+- :link: Openmaptiles Github: https://github.com/openmaptiles/openmaptiles
+
+
+Take a look or copy a standard layer like [building](https://github.com/openmaptiles/openmaptiles/tree/master/layers/building) to get started with your own layer.
+A layer consists out of a **Layer** definition written in YAML format.
+
+## Generate tiles with custom layers
+
+1. Download the [openmaptiles](https://github.com/openmaptiles/openmaptiles) main project.
+2. Go into the `openmaptiles` directory
+3. Add the layer to the `openmaptiles.yaml` [more](Add the layer to the Tileset).
+4. Execute make commands:
+    ```
+    make clean                  # Remove the build directory
+    make                        # Generate sql out of the layers
+    make import-osm             # Import OSM Data into the tables they are defined in mapping.yaml
+    make import-sql             # Import SQL functions from the <layer>.sql (park.sql)
+    make generate-tiles-pg      # Generate .mbtiles file
+    ``` 
+5. Start your [Tileserver](https://openmaptiles.org/docs/host/tileserver-gl/). 
+    ```
+    make start-tileserver
+    ```
+    And go to `localhost:8090`
+6. **Optional** for better developing, start a [Maputnik](https://openmaptiles.org/docs/style/maputnik/) server to check the layers data in the **Inspect** mode.
+    ```
+    make start-maputnik
+    ```
+    Which is available over the PORT `8088`: `localhost:8088`
+
+For the complete workflow to generate tiles with the earth data and wikidata look into this: [Workflow to generate tiles](https://github.com/openmaptiles/openmaptiles#workflow-to-generate-tiles)
+
+
+TODO: Explain how to connect maputnik with the local tiles. Is this needed?
+
+## Layer structure    
+
+TODO: A good sample project is needed
+
+To explain the structure, the sample **park** is used.  
+The following layers are childs of the folder **park**:  
+  
+| File name | Description |  
+| --- | --- |  
+| park.yaml | Start point / General definition file. Contains `description`, `fields`, `buffer_size`, `schema`, `datasources`, ...  |  
+| mapping.yaml | [Imposm3 mapping file](https://imposm.org/docs/imposm3/latest/mapping.html). Definition how and where the OSM Data schould be stored in the Database. |  
+| park.sql | Creates SQL functions to get the correct data from the database while tile-generation |  
+  
```

Comment:
A description of the update process is needed here.  Otherwise, tables will only get an initial load if any tables are created beyond the ones that imposm makes.